cognite/client/utils/_identifier.py

```diff
@@ -259,3 +274,35 @@ def load(cls, user_identifiers: str | Sequence[str]) -> UserIdentifierSequence:
     def assert_singleton(self) -> None:
         if not self.is_singleton():
             raise ValueError("Exactly one user identifier (string) must be specified")
+
+
+class WorkflowVersionIdentifierSequence(IdentifierSequenceCore[WorkflowVersionIdentifier]):
```

Comment:
I'm wondering if we can manage to avoid having to reimplement the IdentifierSequence for every new Identifier type.. We can follow up on that though, since these aren't part of the public API.

Reply:
Amen, really needed.